Content.Server/Power/EntitySystems/PowerMonitoringConsoleSystem.cs

```diff
@@ -0,0 +1,101 @@
+using Content.Shared.Popups;
+using Content.Shared.Power;
+using Content.Server.NodeContainer;
+using Content.Server.NodeContainer.Nodes;
+using Content.Server.Power.Components;
+using Content.Server.Power.NodeGroups;
+using Content.Server.UserInterface;
+using Content.Server.WireHacking;
+using JetBrains.Annotations;
+
+namespace Content.Server.Power.EntitySystems;
+
+[UsedImplicitly]
+internal sealed class PowerMonitoringConsoleSystem : EntitySystem
+{
+    private float _updateTimer = 0.0f;
+
+    public override void Update(float frameTime)
+    {
+        _updateTimer += frameTime;
+        if (_updateTimer >= 1)
+        {
+            _updateTimer -= 1;
```

Comment:
the 1 here should be a static/const in the system

Reply:
Fixed in ac07a2b5d932ee335d2eb99c6105a3eec0024cb6